delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/DeleteProject.scala

```diff
@@ -0,0 +1,78 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews
+
+import akka.actor.ActorSystem
+import akka.persistence.cassandra.cleanup.Cleanup
+import akka.persistence.query.NoOffset
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.BlazegraphViews
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.model.BlazegraphViewSearchParams
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeViewSearchParams
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.ElasticSearchViews
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewSearchParams
+import ch.epfl.bluebrain.nexus.delta.sdk.Projects
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Envelope
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Event.ProjectScopedEvent
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceF.defaultSort
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.{ResourceCommand, ResourceEvent, ResourceRejection, ResourceState}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.Pagination.OnePage
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
+import ch.epfl.bluebrain.nexus.delta.sourcing.{Aggregate, EventLog}
+import monix.bio.{IO, Task}
+
+import scala.concurrent.duration._
+
+class DeleteProject(
```

Comment:
I don't think I understand why this is inside composite views? Isn't deleting elasticsearch and blazegraph handled already their respective `ViewsDeletion`s?

Reply:
This was just a test I did previously. You are right, I forgot to delete it.